### PR TITLE
fix: talking heads for mobile, css fixes, forward message modal fix

### PR DESCRIPTION
**Pull request by iqmulator**

### CODE REVIEW

File: src/dotnet/Chat.UI.Blazor/Components/Banners/TalkingMembersBanner.razor [hunk 1]
```diff
@@ -0,0 +1,88 @@
+@namespace ActualChat.Chat.UI.Blazor.Components
+@inherits ComputedStateComponent<TalkingMembersBanner.Model>
+@{
+    var m = State.Value;
+    if (ReferenceEquals(m, Model.None))
+        return;
+
+    if (!ScreenSize.IsNarrow())
```
Comment (crui3er) [inline src/dotnet/Chat.UI.Blazor/Components/Banners/TalkingMembersBanner.razor]: Better to put this check to Banners.razor and do not render TalkingMembersBanner on wide screens.

File: src/dotnet/Chat.UI.Blazor/Components/Banners/TalkingMembersBanner.razor [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+            <Button Class="btn-primary talking-banner-btn" Click="@OnListenClick">
+                <Icon><i class="icon-listen text-xl"></i></Icon>
+                <Title>Listen</Title>
+            </Button>
+        }
+    </Buttons>
+</Banner>
+
+@code {
+    private Session Session => ChatContext.Hub.Session();
```
Comment (crui3er) [inline src/dotnet/Chat.UI.Blazor/Components/Banners/TalkingMembersBanner.razor]: Extract Hub property to make accessing services shorter:
private ChatUIHub Hub => ChatContext.Hub;
private AccountUI AccountUI => Hub.AccountUI;

File: src/dotnet/Chat.UI.Blazor/styles.css [hunk 3]
```diff
@@ -22,6 +23,7 @@
 @import './Components/DateVisor/date-visor.css';
 @import './Components/Discover/discover.css';
 @import './Components/ForwardMessage/forward-message-modal.css';
+@import './Components/HIstoricalPlaybackSubHeader/historical-playback-subheader.css';
```
Comment (crui3er) [inline src/dotnet/Chat.UI.Blazor/styles.css]: Rename directory 'HIstoricalPlaybackSubHeader' -> 'HistoricalPlaybackSubHeader'

**Review by crui3er — CHANGES_REQUESTED** [review]

File: src/dotnet/Chat.UI.Blazor/styles.css (hunk 3 above)
Comment (iqmulator) [reply src/dotnet/Chat.UI.Blazor/styles.css]: Oh...

**Review by crui3er — APPROVED** [review]

**Pull request merged**